packages/manager/src/features/Databases/DatabaseCreate/DatabaseCreate.tsx

```diff
@@ -459,17 +461,26 @@ const DatabaseCreate = () => {
       'cluster_size',
       values.cluster_size < 1 ? 3 : values.cluster_size
     );
-    setFieldValue(
-      'replication_type',
-      determineReplicationType(values.cluster_size, values.engine)
-    );
-    setFieldValue(
-      'replication_commit_type',
-      determineReplicationCommitType(values.engine)
-    );
+    !isDatabasesV2Enabled &&
+      setFieldValue(
+        'replication_type',
+        determineReplicationType(values.cluster_size, values.engine)
+      );
```

Comment:
Rather than doing `&&`, I think it would be more explicit and readable to use if statements 

```suggestion
     if (!isDatabasesV2Enabled) {
        setFieldValue(
          'replication_type',
          determineReplicationType(values.cluster_size, values.engine)
        );
      }
```